Models/Rom/MessageEntry.cs

```diff
@@ -1,10 +1,31 @@
-﻿
+﻿using System;
+
 namespace MMRando.Models.Rom
 {
     public class MessageEntry
     {
         public ushort Id;
-        public int Address;
-        public int Size;
+        public byte[] Header = new byte[11];
+        public string Message;
+
+        /// <summary>
+        /// For MessageTable access only
+        /// </summary>
+        public int address;
+
+
+        public int Size => (Message.Length + Header.Length + 3) & -4;
```

Comment:
Does this just round it down the nearest 4 bits?

Reply:
It rounds up to the nearest 4 bytes. and yes, as long as length is a positive integer.